modules/javafx.controls/src/main/java/javafx/scene/control/skin/TextInputControlSkin.java

```diff
@@ -334,7 +334,10 @@ protected boolean computeValue() {
         control.setInputMethodRequests(new ExtendedInputMethodRequests() {
             @Override public Point2D getTextLocation(int offset) {
                 Scene scene = getSkinnable().getScene();
-                Window window = scene.getWindow();
+                Window window = scene != null ? scene.getWindow() : null;
+                if (window == null) {
+                    return new Point2D(0, 0);
+                }
```

Comment:
```suggestion
                if (scene == null) {
                    return new Point2D(0, 0);
                }
                Window window = scene.getWindow();
```

Reply:
Since the NullPointer occurs here because the synchronization between JavaFx and AWT threads is not really possible in this case, we cannot ensure that this method is only called when the scene is assigned to a window.
I would check here for null both the scene and the window.

Reply:
What a pity. Then ignore this comment.